index.html

```diff
@@ -980,51 +980,11 @@ <h2>Status</h2>
         <h2>Presentations</h2>
 
         <p>
-Credentials MAY be composed by placing them into a data structure called a
-<a>presentation</a>. A <a>verifiable presentation</a> is a presentation that contains
-<a>verifiable credentials</a> and one or more proofs that are appropriate
-for the presentation. The basic structure of a <a>verifiable presentation</a>
-is provided below:
+Credentials MAY be used to make a <a>verifiable presentation</a>. The presentation
+may be of a credential itself, or of a profile, or of data derived from credentials
+in a format that is cryptographically verifiable.
         </p>
 
-        <pre class="example nohighlight" title="Basic structure of a presentation">
-{
-  "@context": "https://w3id.org/credentials/v1",
-  "id": "urn:uuid:3978344f-8596-4c3a-a978-8fcaba3903c5",
-  "type": ["VerifiablePresentation"],
-  <span class="highlight">"verifiableCredential": [{ ... }],
-  "proof": [{ ... }]</span>
-}
-        </pre>
-
-        <p class="issue" data-number=133>
```

Comment:
We can't remove issues from the spec that are not resolved.

Reply:
this was a mistake on my part. Will revise.